src/chains/details/avalanche.ts

```diff
@@ -0,0 +1,45 @@
+import { nativeCurrencyTemplate } from '../../common/consts/tokens'
+import { ChainInfo, SupportedChainId } from '../types'
+import { RAW_CHAINS_FILES_PATH } from '../const/path'
+
+const avalancheLogo = `${RAW_CHAINS_FILES_PATH}/images/avalanche-logo.svg`
```

Comment:
Add a TODO about the logo?
As you mentioned, currently it's the AVAX token logo, not the chain's